src/sqs/index.ts

```diff
@@ -258,23 +314,66 @@ export class SqsQueue<MessagePayload> {
 			executionId: uuidv4(),
 			queue: this.queueName})
 
-		const context: Context<MessagePayload> = {message, payload, log }
+		const maybeReceivedCount: MessageAttributeValue | undefined = message.MessageAttributes ? message.MessageAttributes["ApproximateReceiveCount"] : undefined;
```

Comment:
use optional chaining.

Reply:
I tried but it doesn't work
![image](https://user-images.githubusercontent.com/42425541/141011938-bfb9aa11-6bb1-46df-a929-30b87934b9f9.png)

Reply:
That's not the syntax, you're missing a dot.  `message.MessageAttributes?.["ApproximateReceiveCount"]` but you don't even need the string accessor, you can just do  `message.MessageAttributes?.ApproximateReceiveCount` directly.